### DISCUSSION TITLE
native-like min/max attributes?

### DISCUSSION THREAD

**Discussion by FezVrasta** [gpbl/react-day-picker#2506] [Support] [answered]
Hi, I can't find a way to define a min/max set of attributes that work like the native input date picker. 

- https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/min
- https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max

Is it not supported?

**Comment by wilsonadenuga** [comment]
> Hi, I can't find a way to define a min/max set of attributes that work like the native input date picker.
> 
> * https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/min
> * https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max
> 
> Is it not supported?

it is supported @FezVrasta , check 

[https://daypicker.dev/docs/navigation#start-and-end-dates](https://daypicker.dev/docs/navigation#start-and-end-dates)

**Reply by FezVrasta** [reply]
Thanks, does it only support months and not specific dates?

**Reply by wilsonadenuga** [reply]
can you give an example of what you are trying to achieve @FezVrasta

**Reply by FezVrasta** [reply]
`<input type="date" min="2024-10-01" max="2024-10-10" />`

**Reply by wilsonadenuga** [reply, marked as answer]
```
const minDate = new Date('2024-10-01');
const maxDate = new Date('2024-10-10');

<DayPicker
	mode="single"
	startMonth={new Date('2024-10-01')}
	endMonth={new Date('2024-10-10')}
	hidden={{
		before: minDate,
		after: maxDate,
	}}
			
/>
```

**Reply by FezVrasta** [reply]
Thanks!
